alpha/scripts/utils/migrationWebcastUpdateMakeHiddenAndDisplayInSearch.php

```
@@ -0,0 +1,109 @@
+<?php
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+if (count($argv) < 2)
+{
+    die('pleas provide partner id as input' . PHP_EOL .
+        'to run script: ' . basename(__FILE__) . ' X' . PHP_EOL .
+        'whereas X is the path to the partner ids file' . PHP_EOL);
+}
+
+$dryRun = !(count($argv) == 3 && @$argv[2] == "realrun");
```

Don't use @ since it silences error if there are any.